### ISSUE TITLE
http:// urls generated for https:// site, APP_TRUSTED_PROXIES doesn't work

### ISSUE THREAD

**Issue by neunenak** [monicahq/monica#5954]
**Describe the bug**

I'm seeing exactly the same issue as https://github.com/monicahq/monica/issues/4935  - that is, I'm running monica via docker-compose on one (non-public) server, exposing a random port to monica, and running a public-facing nginx on another server that handles SSL and proxies to monica; and monica is generating some `http://` not `https://` URLs on the front page.

My nginx config has the lines:
```
                        proxy_redirect http:// https://;
                        proxy_set_header Host $host;
                        proxy_set_header X-Real-IP 192.168.1.100;
                        proxy_set_header X-Forwarded-For 192.168.1.100;
                        proxy_set_header X-Forwarded-Proto $scheme;
```
I have `APP_TRUSTED_PROXIES="*"` in my .env file for the docker setup. The page HTML has links that look like:

`<link rel="stylesheet" href="http://my.monica.instance.example.com/css/app-ltr.css?id=523470c277b241ddce13">`
 
 
 with no https. I'm not sure what else to try here.

**Comment by asbiin** [comment]
Thank you, the trust_proxies property has indeed been removed, I'll fix it.

**Comment by neunenak** [comment]
@asbiin I upgraded to the latest docker image but am still seeing the same problem. I tried changing the variable in `.env` to `APP_TRUST_PROXIES` since that's what it looks like the patch expects: https://github.com/monicahq/monica/pull/5955/files#diff-950397cc5ef29707b54d0c774bd94af85836bab1a27f2f86533d832d25caf224R152 . But that didn't work either. Maybe there should be a way to explicitly declare what the public name of a monica instance is in the configuration, so the app doesn't need to try and figure it out from the headers? It would be nice if the already-existing `APP_URL` key just did that.
